SmartDeviceLinkTests/UtilitiesSpecs/Touches/SDLTouchManagerSpec.m

```diff
@@ -85,15 +87,26 @@ + (void)testTouchesWithTimeout:(CGFloat)timeoutTime
 QuickSpecBegin(SDLTouchManagerSpec)
 
 describe(@"SDLTouchManager Tests", ^{
-    __block SDLTouchManager *touchManager;
-
+    __block SDLTouchManager *touchManager = nil;
+    __block void (^unloadTouchManager)(void) = ^() {
```

Comment:
This brakes tests. I'd better keep it as it is.
BTW: I tried to add to SDLTouchManager.m the following method though it did not help:
-(void)dealloc { [[NSNotificationCenter defaultCenter] removeObserver:self]; }

Besides SDL supports iOS-8 and up. In iOS-8 I must unsubscribe from notifications otherwise it crashes when calling on zombie object.